oe_theme.theme

```diff
@@ -521,7 +521,17 @@ function oe_theme_preprocess_paragraph__oe_list_item(array &$variables): void {
   /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
   $paragraph = $variables['paragraph'];
 
-  $variables['variant'] = $paragraph->get('field_oe_list_item_variant')->first()->value;
+  // @todo: Fix paragraphs with the new variants.
+  $variants = [
+    'list_item_default' => 'default',
+    'list_item_highlight' => 'highlight',
+    'list_item_date' => 'date',
+    'list_item_thumbnail_primary' => 'thumbnail',
+    'list_item_thumbnail_secondary' => 'thumbnail_secondary',
+  ];
+
+  $paragraph_variant = $variants[$paragraph->get('field_oe_list_item_variant')->first()->value];
```

Comment:
I have not implemented that because this is subject to change in the followup ticket when we address the paragraph implementation. Now these values are predefined configurations coming from the paragraph module, so I don't see possibility to have a non defined variant passed, since this array was created based on those configs temporarily.

Reply:
OK. got it